src/pl/senla/lecture3/task3/product/ProductImpl.java

```diff
@@ -0,0 +1,52 @@
+package pl.senla.lecture3.task3.product;
+
+import pl.senla.lecture3.task3.part.IProductPart;
+
+public class ProductImpl implements IProduct{
+
+    private final IProductPart firstProductPart;
+    private final IProductPart secondProductPart;
+    private final IProductPart thirdProductPart;
+
+    public ProductImpl(IProductPart firstProductPart, IProductPart secondProductPart, IProductPart thirdProductPart) {
+        this.firstProductPart = firstProductPart;
+        this.secondProductPart = secondProductPart;
+        this.thirdProductPart = thirdProductPart;
+    }
+
+    public IProductPart getFirstProductPart() {
+        return firstProductPart;
+    }
+
+    public IProductPart getSecondProductPart() {
+        return secondProductPart;
+    }
+
+    public IProductPart getThirdProductPart() {
+        return thirdProductPart;
+    }
+
+    @Override
+    public void installFirstPart(IProductPart firstProductPart) {
```

Comment:
And in this methods lets install parts inside class attributes, because right now, parts installed through constructor and they only writing logs. More logically will be creating empty product, and installing parts through this methods